custom-namespaces.yaml

```diff
@@ -732,6 +736,11 @@ firefox_desktop:
       tables:
         - table: mozdata.default_browser_agent.default_browser
   explores:
+    urlbar_events:
+      type: events_explore ???
+      views:
+        base_view: urlbar_events ???
+        extended_view: ???
```

Comment:
```suggestion
    urlbar_events:
      type: events_explore
      views:
        base_view: urlbar_events
        extended_view: urlbar_events_table
```

Reply:
Here, the `urlbar_events` explore is of type `events_explore`, and the explore utilizes the `urlbar_events` Looker view (some of the confusion here is b/c Looker separates explores and views, and views are _also_ a thing in BQ).

Reply:
@fbertsch thanks for explaining!